src/Lykke.AlgoStore.Services/AlgoInstancesService.cs

```diff
@@ -353,5 +357,49 @@ ILog log
             var algoInstances = await _instanceRepository.GetAllByWalletIdAndInstanceStatusIsNotStoppedAsync(walletId);
             return algoInstances != null && algoInstances.Any();
         }
+
+        private void ValidateInstanceMetadataDates(AlgoMetaDataInformation instanceMetadata)
+        {
+            var dtType = typeof(DateTime).FullName;
+
+            var instanceParameters = instanceMetadata.Parameters.Where(p => p.Type == dtType).ToList();
+            var startFromDate =  instanceParameters.SingleOrDefault(t => t.Key == "StartFrom")?.Value;
+            var endOnDate = instanceParameters.SingleOrDefault(t => t.Key == "EndOn")?.Value;
+
+            var instanceStartFromDate = DateTime.ParseExact(startFromDate, "yyyy-MM-ddTHH:mm:ss.fffZ", CultureInfo.InvariantCulture,
```

Comment:
Shouldn't the datetime format be a constant?

Reply:
Yes, thanks. Will change that right away